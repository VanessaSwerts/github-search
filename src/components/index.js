import CustomHeader from './Header/Header'
import CustomFooter from './Footer/Footer'
import Error from './Error/Error'
import Loading from './Loading/Loading'

export {
  CustomHeader,
  CustomFooter,
  Error,
  Loading
}