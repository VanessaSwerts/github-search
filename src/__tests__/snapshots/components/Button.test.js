import React from 'react'
import { render } from '@testing-library/react'
import { Button } from '../../../components'

describe('Testes de Snapshot da Button', () => {

  test('Renderização do componente Button', () => {
    const { asFragment } = render(<Button />)

    expect(asFragment(<Button />)).toMatchSnapshot()
  })

}) 