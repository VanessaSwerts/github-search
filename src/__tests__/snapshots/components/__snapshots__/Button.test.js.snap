// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de Snapshot da Button Renderização do componente Button 1`] = `
<DocumentFragment>
  <button
    class="button"
    data-testid="search-button"
    type="submit"
  />
</DocumentFragment>
`;
